### PR TITLE
Some new information. :-)

### PR DESCRIPTION
**Pull request by zoltancsonka**
My first pull request.

### CODE REVIEW

File: _posts/0000-01-06-zoltancsonka.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-![bad-luck-brian](https://cloud.githubusercontent.com/assets/16547949/25400743/8ae447a2-29c1-11e7-9839-5438d8fcda7b.jpg)
+![morpheus](https://cloud.githubusercontent.com/assets/16547949/25400958/410316ee-29c2-11e7-9e6d-9d13999e76e3.jpg)
 {: .center}
 
-Caption here
+Hi there....
```
Comment (ghost) [inline _posts/0000-01-06-zoltancsonka.md]: Please change the caption

**Review by ghost — CHANGES_REQUESTED** [review]
Please change the caption

File: _posts/0000-01-06-zoltancsonka.md (hunk 1 above)
Comment (zoltancsonka) [reply _posts/0000-01-06-zoltancsonka.md]: Ok.

**Review by ghost — APPROVED** [review]
ok

**Pull request merged**